src/parser/monk/mistweaver/modules/spells/Vivify.js

```diff
@@ -41,12 +41,10 @@ class Vivify extends Analyzer {
       this.gustsHealing += (event.amount || 0) + (event.absorbed || 0);
     }
 
-    if (SPELLS.VIVIFY.id !== spellId || this.lastCastTarget === event.targetID) {
```

Comment:
Unfortunately the old Vivify tracker code was not allowing for the proper tracking of gustsHealing. I think the new logic should provide old ReM vivify tracking along with gusts tracking.